playbooks/roles/mongodb/tasks/main.yml

```diff
@@ -135,5 +146,12 @@
       copy:
         content: "{{ MONGODB_EXPORTER_CONSUL_SERVICE | to_nice_json }}"
         dest: /etc/consul/mongodb-exporter.json
-      notify:
-        - reload consul
+      notify: reload consul
+
+
+    - name: Install PIP later needed for greendayonfire.mongodb
+      apt:
+        pkg:
+          - python-dev
+          - python-pip
+          - python3-pip
```

Comment:
Remove this section, it's already included in `common-server`

Reply:
`python-dev` can be removed (I removed it but if we remove python3-pip it would error out in greendayonfire.mongodb playbook